Converters/DecimalConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace InvestingOak.Converters
+{
+    public class DecimalConverter : JsonConverter<decimal>
+    {
+        public override decimal Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.String)
+                if (decimal.TryParse(reader.GetString(), out decimal d))
+                    return d;
+
+            return default;
+        }
+
+        public override void Write(Utf8JsonWriter writer, decimal value, JsonSerializerOptions options)
+        {
+            writer.WriteStringValue(value.ToString());
```

Comment:
Codacy found an issue: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6348574)